src/components/dialogs/create-team.tsx

```diff
@@ -0,0 +1,85 @@
+import { WalletIcon } from "@heroicons/react/20/solid";
+import { toast } from "react-toastify";
+import { z } from "zod";
+import { Button } from "~/components/ui/button";
+import { useZodForm } from "~/hooks/form";
+import { useCreateTeam, useListTeams } from "~/hooks/teams";
+import usePersistingStore from "~/store/useStore";
+import { TextInput } from "../inputs/text";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from "../ui/dialog";
+import { Form } from "../ui/form";
+
+const formSchema = z.object({
+  name: z.string(),
+});
+
+export function CreateTeamDialog({
```

Comment:
Usually, the component and file name are same (PascalCase) -  i.e CreateTeamDialog and CreateTeamDialog.tsx. 
Only pages folder contains lower case file names as it's NextJS Routing feature - navigation based on file names